### DISCUSSION TITLE
The generated PPTX file cannot be opened using office and cannot be edited

### DISCUSSION THREAD

**Discussion by ghost** [marp-team/marp#512] [Q&A] [answered]
> [!NOTE]
> @moguixiaolin _This post has been translated to English by the Marp team._ For the community's discoverability and workability, please your help in writing the description in English. You can also use a translator.

<details>
<summary>Original</summary>

### 生成的PPTX文件无法使用office打开无法编辑

markdown文件，通过NPM指令：marp --pptx slide-deck.md
生成出的ppt文件，无法在officez中进行编辑操作

</details>

---

markdown file, through NPM command: marp --pptx slide-deck.md
The generated ppt file cannot be edited in officez

**Comment by yhatt** [comment, marked as answer]
See #82 in frequently asked category.
